### DISCUSSION TITLE
want to get truee hieght and width of the entry widget

### DISCUSSION THREAD

**Discussion by plugeit** [TomSchimansky/CustomTkinter#429] [Q&A]
for example : If i have a entry widget entry=ctk.CTkEntry(screen,hieght=20,width=20,font='air 15 bold')
now the widget readjust itself to different height and width and i want to know its height and width after readjustment but i can't so how to do it

** if use entry['height'] entry['width'] it not gives the correct value.

Please Help.

**Comment by avalon60** [comment]
Have you tried using something like:

root.update_idletasks()     (change root accordingly to your master window)
width = widget.winfo_width()
height = widget.winfo_height()
